pkg/stub/handler.go

```diff
@@ -1624,6 +1629,24 @@ func (g *defaultConfigMapClientWrapper) Delete(namespace, name string) error {
 	return g.h.coreclient.ConfigMaps(namespace).Delete(name, &metav1.DeleteOptions{})
 }
 
+type defaultConfigClientWrapper struct {
```

Comment:
should have ClusterOperator somewhere in the name since it only operators on ClusterOperator resources?

Reply:
It can't be used generically because your function signatures aren't generic (ie "Get()" can only return a ClusterOperator).

Reply:
Yeah but I can create new interfaces that leverage configclient but call something in place of `ClusterOperators()`

Reply:
it's a struct, how are you going to create new interfaces?  If you want additional flavors of Get() you'd have to define another struct.

Reply:
I think I got hung up on the wrong line .... thought you were talking about the h.configclient and not the struct  (stupid forgetting my reading glasses today ;-)

I'll rename the struct